src/hessian.F90

```diff
@@ -234,11 +242,19 @@ subroutine numhess( &
 #endif
    endif
 
+   endif !From turbomole_exception
+
+
 !  Hessian done -----------------------------------------------------------
 !! ========================================================================
 
    if (runtyp.eq.p_run_bhess) call numhess_rmsd(env,mol,hbias)
 
+   if(mode_extrun .eq. p_ext_turbomole .AND. runtyp.eq.p_run_bhess) then 
+        htb = h !htb equals unbiased hessian
```

Comment:
Is this assignment here required, `htb` will be overwritten by assignment later.